code/datums/elements/decal.dm

```diff
@@ -4,15 +4,26 @@
 	var/cleanable
 	var/description
 	var/mutable_appearance/pic
-
-/datum/element/decal/Attach(atom/target, _icon, _icon_state, _dir, _cleanable=FALSE, _color, _layer=TURF_LAYER, _description, _alpha=255)
+	/**
+	 *  A short lecture on decal element collision on rotation
+	 *  If a given decal's rotated version is identical to one of existing (at a same target), pre-rotation decals,
+	 *  then the rotated decal won't stay after when the colliding pre-rotation decal gets rotated,
+	 *  resulting in some decal elements colliding into nonexistence. This internal tick-tock prevents
+	 *  such collision by forcing a non-collision.
+	 */
+	var/rotated
```

Comment:
Is this rotated var ever used? I can only find cases where it is assigned to, but the value itself doesn't seem to be used apart from to assign to itself again.

Reply:
Ah I see why this is used.